Terasology/LightAndShadow
# feat: add a UIImage next to the answer UIButton

### Tasks
- [x] set ColumnLayout for the UIImages

### Purpose

Set an arrow asset to appear in the appropriate UIImage next to the hoved UIButton.

### How to test

- Check https://github.com/Terasology/LightAndShadowResources/pull/53
- Check https://github.com/Terasology/Dialogs/pull/15
- Start a LAS game
- Talk to The Fool NPC
- See the image/s next to the answer button/s

_note: the extra response and large text at the dialogue box is not committed and is not planned to be_
![image](https://user-images.githubusercontent.com/48293545/90340862-a66e6d80-e003-11ea-9703-096ab73dbb79.png)
